app/models/post.rb

```diff
@@ -0,0 +1,21 @@
+class Post < ActiveRecord::Base
+  belongs_to :author, class_name: 'User', foreign_key: 'authors_id'
+  has_many :likes, class_name: 'Like'
+  has_many :comments, class_name: 'Comment'
+
+  def recent_comments
+    Comment.where(posts_id: id).limit(5).order(created_at: :asc).pluck(:text)
+  end
+
+  def update_comments_counter
+    post = self
+    post.increment!(:comments_counter)
+    post.save
+  end
```

Comment:
- [ ] As stated in the project requirement **A method that updates the comments counter for a post should be Comment model instead of post Model**  So Kindly move to Comment model and also implemented the below optional suggestion.
- [ Optional ] Good Job for using [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save` to call the `#update_comments_counter` method every time a new comment has been saved to database, then **Kindly I recommend you to make the `update_comments_counter` method private**.